sru/cuda_functional.py

```diff
@@ -4,8 +4,8 @@
 
 from torch.utils.cpp_extension import load
 sources = [
-    os.path.join(os.path.dirname(__file__), "sru_cuda_impl.cpp"),
-    os.path.join(os.path.dirname(__file__), "sru_cuda_kernel.cu"),
+    os.path.join(os.path.dirname(__file__), os.path.join("csrc", "sru_cuda_impl.cpp")),
+    os.path.join(os.path.dirname(__file__), os.path.join("csrc", "sru_cuda_kernel.cu")),
```

Comment:
You can actually put these all into a single call to join:

```
os.path.join(os.path.dirname(__file__),, 'csrs', 'sru_cuda_kernel.cu')
```
```

Reply:
oh good to know. Will do.